client/setupProxy.js

```diff
@@ -0,0 +1,5 @@
+const proxy = require("http-proxy-middleware");
```

Comment:
Has this been tested on Heroku? @BlinkVonDoom

Reply:
_making a test instance_